.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,10 +1,36 @@
 ---
-name: Feature request
-about: Suggest an idea for this project
+name: Feature Request
+about: Suggest an idea that should be added to the language.
```

Comment:
ideas are not added to the language :)

should be "Suggest a feature that should be added to the language."